src/qt/gui_tabRewards.py

```diff
@@ -36,7 +36,7 @@ def initRewardsForm(self):
         self.mnSelect = QComboBox()
         self.mnSelect.setToolTip("Select Masternode") 
         hBox.addWidget(self.mnSelect)
-        label = QLabel("Address Balance")
+        label = QLabel("Total Balance")

```

Comment:
`Total` could be misleading here.
One could think it's the total balance of a masternode (collater + rewards).

For example, creating 2 masternodes with the same collateral address is perfectly legal (provided they have different UTXOs).